### PR TITLE
Stix 1x

### PR DESCRIPTION
**Pull request by rpiazza**
Examples of extension definitions.

Used in the elevator

### CODE REVIEW

File: extension-definition-specifications/stix-1x/stix1x-network-socket.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+			"type": "string"
+		},
+	        "local_address": {
+			"description": "Specifies the IP address and port for the socket on the local machine",
+			"type": "string"
+		},
+	        "remote_address": {
+			"description": "Specifies the IP address and port for the socket on the remote machine",
+			"type": "string"
+		}
```
Comment (clenk) [inline extension-definition-specifications/stix-1x/stix1x-network-socket.json:25]: These properties could probably have regex patterns associated with them.

**Review by clenk — COMMENTED** [review]
Do you wanna change the `$id`s to the oasis-open repo instead of rpiazza?

File: extension-definition-specifications/stix-1x/stix1x-coa.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+		    ]
+		},
+		"stage": {
+		    "type": "string",
+		    "description": "open vocab, suggested values in #/deinitions/coa-stage-ov"
+		    
+		},
+		"objective": {
+		    "type": "string",
+		    "description": ""
```
[the remark this answers is not in the document]
Comment (rpiazza) [reply extension-definition-specifications/stix-1x/stix1x-coa.json:21]: Done

File: extension-definition-specifications/stix-1x/stix1x-network-socket.json (hunk 1 above)
Comment (rpiazza) [reply extension-definition-specifications/stix-1x/stix1x-network-socket.json:25]: protocol_family is really an enum - so I changed it.  local/remote address should really be ip address SCOs, and then be used as values for src_ref/dst_ref.  Made that change also.

**Pull request merged**